### ISSUE TITLE
describe your audience?

### ISSUE THREAD

**Issue by tamsky** [metrics20/spec#4]
What population comprises your target audience for the metrics20.org website and the spec called "Metrics 2.0"?

**Comment by Dieterbe** [comment]
Its aimed for monitoring software developers and power users

**Comment by tamsky** [comment]
Thanks for the clarification.

From my reading here, the metrics20/spec seems very closely tied to graphite.

I'm having a bit of trouble understanding how it could apply to other monitoring software projects (OpenTSDB/KairosDB, Prometheus, InfluxDB, druid, etc.), which do things "differently"...

**Comment by Dieterbe** [comment]
> From my reading here, the metrics20/spec seems very closely tied to graphite.

you're right. this is because the project started out as an extension to graphite and most of the current tooling works best or only with graphite.

In the spec, I think we should just take out the "carbon protocol" stuff since that's just an example implementation for the graphite-centered setups. we can list protocols somewhere else as examples but not part of the spec.

does that cover it, or do you see other cases that are too tied to graphite itself?

The projects you mention should work with metrics2.0, since they all support tags in one form or another. I think the current enforcement of "metrics should only be identified as a set of key value pairs" is too harsh and it should probably be a key + a set of key/value pairs, which is how all the projects you mention do it.
thoughts? any other reason why you think it wouldn't work well with those projects?

thanks for your input! The projects needs fresh eyes like yours :)

**Comment by Dieterbe** [comment]
2fa3265a7735bcf0590e2121144b8d55d86bab91 takes out graphite ties out of the spec.
